crds/virtualmachines.yaml

```diff
@@ -0,0 +1,1660 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  labels:
+    heritage: deckhouse
+    module: virtualization
+  name: virtualmachines.virtualization.deckhouse.io
+spec:
+  group: virtualization.deckhouse.io
+  names:
+    categories:
+      - all
+      - virtualization
+    kind: VirtualMachine
+    listKind: VirtualMachineList
+    plural: virtualmachines
+    shortNames:
+      - vm
+      - vms
+    singular: virtualmachine
+  scope: Namespaced
+  versions:
+    - additionalPrinterColumns:
+        - description: The phase of the virtual machine.
+          jsonPath: .status.phase
+          name: Phase
+          type: string
+        - description: The number of cores of the virtual machine.
+          jsonPath: .spec.cpu.cores
+          name: Cores
+          type: string
+        - description: Virtual machine core fraction.
+          jsonPath: .spec.cpu.coreFraction
+          name: CoreFraction
+          type: string
+        - description: The amount of memory of the virtual machine.
+          jsonPath: .spec.memory.size
+          name: Memory
+          type: string
+        - description: A restart of the virtual machine is required.
+          jsonPath: .status.conditions[?(@.type=='AwaitingRestartToApplyConfiguration')].status
+          name: Need restart
+          type: string
+        - description: Agent status.
+          jsonPath: ..status.conditions[?(@.type=='AgentReady')].status
+          name: Agent
+          type: string
+        - description: The node where the virtual machine is running.
+          jsonPath: .status.nodeName
+          name: Node
+          type: string
+        - description: The IP address of the virtual machine.
+          jsonPath: .status.ipAddress
+          name: IPAddress
+          type: string
+        - description: Time of creation resource.
+          jsonPath: .metadata.creationTimestamp
+          name: Age
+          type: date
+      name: v1alpha2
+      schema:
+        openAPIV3Schema:
+          description: VirtualMachine specifies configuration of the virtual machine.
```

Comment:
you missed this
https://github.com/deckhouse/virtualization/blob/main/crds/virtualmachine.yaml#L30-L36

please synchronize the document with the original